ui/app/AppLayouts/Communities/stores/CommunitiesStore.qml

```diff
@@ -230,6 +237,14 @@ QtObject {
         return communitiesModuleInst.isDisplayNameDupeOfCommunityMember(displayName)
     }
 
+    function leaveCommunity(communityId) {
+        d.profileSectionModuleInst.communitiesModule.leaveCommunity(communityId)
```

Comment:
Going through the profile module to call a community function sounds wrong.

We can move the `leaveCommunity` from the profile module to the community module in nim and use `communitiesModuleInst` instead